cellxgene_resources/upload_local.ipynb

```diff
@@ -26,8 +26,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "scc_repo_loc = os.path.expanduser('~/GitHub/CZI/')\n",
-    "api_key_file_path = os.path.expanduser('~/Documents/cxg-api-key-dev.txt')"
+    "scc_repo_loc = os.path.expanduser('~/GitClones/Lattice-Data/')\n",
+    "api_key_file_path = os.path.expanduser('~/Documents/keys/cxg-api-key-dev.txt')"
```

Comment:
I think we'll be using production 90% of the time, so let's default to production keys/site
so `cxg-api-key.txt`

and then remove the `env='dev'` below